plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -16,16 +16,18 @@
 import com.google.common.collect.ImmutableMap;
 import io.trino.testing.QueryRunner;
 
+import static io.trino.plugin.mongodb.MongoQueryRunner.createMongoClient;
 import static io.trino.plugin.mongodb.MongoQueryRunner.createMongoQueryRunner;
 
-public class TestMongoDistributedQueries
-        extends BaseMongoDistributedQueries
+public class TestMongoConnectorTest
+        extends BaseMongoConnectorTest
 {
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        MongoServer server = closeAfterClass(new MongoServer());
+        server = closeAfterClass(new MongoServer());
+        client = createMongoClient(server);
```

Comment:
closeAfterClass?

Reply:
We could, but there is `destroy` method in the base class. I wouldn't touch as it relates to the test migration.